data/Yang_2023/metadata.yml

```diff
@@ -155,10 +160,18 @@ contexts:
     description: open habitat conditions
   - value: overgrown
     description: overgrown habitat conditions
+- context_property: time to flowering type
```

Comment:
Does this need to be mapped under the trait metadata? (I might be confused about this but sorting it out with Lizzy on Slack)

Reply:
We figured that this context property can't be mapped to a single trait, even though it's irrelevant to the other traits, so leaving it as is.